custom/mgh_epic/tasks.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+from celery.schedules import crontab
+
+from corehq.apps.celery import periodic_task
+
+from custom.mgh_epic.sync_epic_appointments import sync_all_appointments_domain
+
+MGH_EPIC_DOMAINS = settings.CUSTOM_DOMAINS_BY_MODULE['custom.mgh_epic']
+
+
+@periodic_task(run_every=crontab(hour="*", minute=1), queue="background_queue")
+def sync_all_epic_appointments():
+    for domain in MGH_EPIC_DOMAINS:
+        try:
+            sync_all_appointments_domain(domain)
+        except Exception as e:
+            raise Exception(f"Error syncing appointments for domain {domain}: {e}")
```

Comment:
Re-raising another exception doesn't really help here. You should rather log it:
```suggestion
            logger.exception("Error syncing epic appointments", extra={"domain": domain})
```

Reply:
changed here: https://github.com/dimagi/commcare-hq/pull/35125/commits/e4653c34506573f8b64a713830af2c483923b0c4